wetterdienst/data_storing.py

```diff
@@ -1,13 +1,21 @@
 """ Data storing/restoring methods"""
+import re
+from io import BytesIO
 from pathlib import Path
-from typing import Union
+from typing import Union, List, Tuple
+import tarfile
+import gzip
+import datetime as dt
```

Comment:
You can load directly datetime from datetime because it is the only part of the datetime package you need.